docs/repo/property-pages/property-specification.md

```diff
@@ -285,6 +285,16 @@ a DynamicEnumProperty instead of a StringProperty. All `SupportedValues` that ar
 will be added as unchecked list items.
 
 #### Example
+The xliff namespace can be defined as follows in your Rule definition. 
+An example can be found in [this property page](https://github.com/dotnet/project-system/blob/main/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml).
```

Comment:
Better to use a permalink here to prevent dead links in future. Such a link would reference the content at a particular commit, rather than the main branch. You can get permalinks by clicking on line numbers in source files on GitHub.

Reply:
To do that, go to the link you provided and press `y` on your keyboard. This will change the link in the address bar to a perma-link with the commit hash in it.

Reply:
That's great, thanks! I'll use that for sure.

Reply:
![image](https://user-images.githubusercontent.com/350947/214442445-e1183bed-bdd5-4022-adf2-c9b25402ad20.png)

Reply:
```suggestion
An example can be found in [this property page](https://github.com/dotnet/project-system/blob/8a7df1bfb0926b9fe93fbc2fb9d1edabcdf917f9/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml).
```

Reply:
We had to provide this guidance to those using our Rest API code generators, as the users of it in the company needed to provide links to their REST specs in our spec GitHub repo. We always needed it as a commit hash so it would generate code consistently.